spec/requests/media_auth_request_spec.rb

```diff
@@ -2,17 +2,11 @@
 
 RSpec.describe "Authentication for Media requests", type: :request do
 
-  let(:allowed_loc) { 'ip.address1' }
   let(:user_no_loc_no_webauth) { User.new }
-  let(:user_no_loc_webauth) { User.new(webauth_user: true) }
```

Comment:
I thought the request specs were sort of the "full stack" spec and it was appropriate to have them here when we had media stream and download actions.   Now that we don't have the media stream action ... where are we asserting these interactions for permissions for streaming requests?    I may be just missing where these interactions are tested in the user spec ... but I have looked there and elsewhere and I'm not seeing the full interactions.

Basically, I may be confused as to where we really are implementing location-based permissions for streams:  sul-embed calls stacks for ?? file read??  and that is where this is now handled for streams?    If so, the file request specs don't handle the case for user webauthed but not stanford ... or doesn't that matter?  Or am I just confused?

I have looked in all the spots you indicated above ... and still feel a bit concerned that we are losing a few test cases along the way.